safebrowser.go

```diff
@@ -338,6 +340,8 @@ func NewSafeBrowser(conf Config) (*SafeBrowser, error) {
 		}
 	}
 
+	sb.nextlookuptime = time.Unix(0,0)
```

Comment:
I was trying to say that we should discard the MinWaitDuration commit for now, and then once the batching is merged, send the MinWaitDuration commit for review separately. Do that and I can merge this now.